templates/documentation/fire.html

```diff
@@ -78,6 +95,22 @@ <h3>Output</h3>
 
 <pre>
 {
+  "aqi_12": {
+    "aqi": &lt;numeric air quality index value&gt;,
+    "pm25_conc": &lt;numeric PM2.5 concentration&gt;
```

Comment:
Does this have a unit that would be relevant to include? Or is it a unit-less index type value like AQI?

Reply:
Good thought Charlie! I added the units used for the PM 2.5 concentration to the latest commit.